guides/common/modules/proc_enabling-rh-cloud-and-insights-client-reports-on-hosts.adoc

```diff
@@ -0,0 +1,20 @@
+[id="enabling-rh-cloud-and-insights-client-reports-on-hosts_{context}"]
+= Enabling RH Cloud and Insights client reports on hosts
+
+You can enable the Red Hat Insights client on hosts and have the hosts send RH Cloud reports to the {RHCloud}.
```

Comment:
```suggestion
You can enable the Red Hat Insights client on hosts and have {Project} upload hosts inventory to the Insights service in the {RHCloud}.
```
I quickly checked in the web UI what exactly is the data that gets sent to RH Cloud. This should be a more precise expression.

Reply:
Thanks, applied.